src/domain/home/moreInfo/MoreInfoLinkList.tsx

```diff
@@ -2,25 +2,26 @@ import { useTranslation } from 'react-i18next';
 
 import styles from './moreInfoLinkList.module.scss';
 import { MoreInfoLink } from './types/moreInfo';
+import { getCurrentLanguage } from '../../../common/translation/TranslationUtils';
 
 type MoreInfoLinkListProps = {
   links: MoreInfoLink[];
 };
 
 const MoreInfoLinkList = ({ links }: MoreInfoLinkListProps) => {
-  const { t } = useTranslation();
+  const { t, i18n } = useTranslation();
+  const currentLanguage = getCurrentLanguage(i18n);
```

Comment:
I don't know the reason why this is getter has been implemented, but `i18n.language` should be enough. I guess the both works.

Reply:
It's used ~10 times otherwise in the codebase:

```typescript
/**
 * Safety check in case
 * somehow i18n have problem or have empty languages.
 */
export const getCurrentLanguage = (i18n: I18nInstanceType): Language =>
  (i18n && i18n.languages && (i18n.languages[0] as Language)) || 'fi';
```

So judging from the code and the comment it's for added robustness.